src/Helper/Helper_PDF_List_Table.php

```diff
@@ -281,15 +281,15 @@ public function column_shortcode( $item ) {
 		$pdf_id = esc_attr( $item['id'] );
 
 		/* Prepare our shortcode sample */
-		$shortcode = '[gravitypdf name="' . esc_attr( $name ) . '" id="' . esc_attr( $item['id'] ) . '" text="' . esc_attr__( 'Download PDF', 'gravity-forms-pdf-extended' ) . '"]';
+		$shortcode  = '[gravitypdf name="' . esc_attr( $name ) . '" id="' . esc_attr( $item['id'] ) . '" text="' . esc_attr__( 'Download PDF', 'gravity-forms-pdf-extended' ) . '"]';
 		$aria_label = sprintf( esc_html__( 'Select to copy the PDF %s shortcode to the clipboard', 'gravity-forms-pdf-extended' ), esc_html( $item['name'] ) );
```

Comment:
We could probably remove the `esc_html` references here, as it gets run through `esc_attr()` below.

Reply:
Copy @jakejackson1 .